cmd/internal/x/security_provider.go

```diff
@@ -1,4 +1,4 @@
-package api
+package x
```

Comment:
Initially in `egoscale/v2`, this file is made obsolete by egoscale v3; however it's still useful for arbitrary calls such as the ones required by x, so it would make sense to move it here.